src/Pim/Bundle/CatalogBundle/spec/Elasticsearch/Filter/IdentifierFilterSpec.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace spec\Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
+
+use Akeneo\Component\StorageUtils\Exception\InvalidPropertyTypeException;
+use PhpSpec\ObjectBehavior;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Filter\AbstractFieldFilter;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Filter\IdentifierFilter;
+use Pim\Bundle\CatalogBundle\Elasticsearch\SearchQueryBuilder;
+use Pim\Component\Catalog\Exception\InvalidOperatorException;
+use Pim\Component\Catalog\Query\Filter\FieldFilterInterface;
+use Pim\Component\Catalog\Query\Filter\Operators;
+use Pim\Component\Catalog\Repository\AttributeRepositoryInterface;
+
+class IdentifierFilterSpec extends ObjectBehavior
+{
+    function let(AttributeRepositoryInterface $attributeRepository)
+    {
+        $this->beConstructedWith(
+            $attributeRepository,
+            ['identifier'],
+            [
+                'STARTS WITH',
+                'CONTAINS',
+                'DOES NOT CONTAIN',
+                '=',
+                '!=',
+                'IN LIST',
+                'NOT IN LIST'
+            ]
+        );
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(IdentifierFilter::class);
+    }
+
+    function it_is_a_field_filter()
+    {
+        $this->shouldImplement(FieldFilterInterface::class);
+        $this->shouldBeAnInstanceOf(AbstractFieldFilter::class);
```

Comment:
As discussed with @aRn0D, it's not useful to precise that we extends an interface, our contract is only to respect the interface which represents our public API :)